arch/arm64/boot/dts/arm/fvp-base-revc.dts

```diff
@@ -243,4 +247,11 @@
 				<0 0 43 &gic 0 0 GIC_SPI 43 IRQ_TYPE_LEVEL_HIGH>,
 				<0 0 44 &gic 0 0 GIC_SPI 44 IRQ_TYPE_LEVEL_HIGH>;
 	};
+
+	firmware {
+		optee {
+			compatible = "linaro,optee-tz";
+			method = "smc";
+		};
+	};
```

Comment:
How was the node added to the DT prior to this PR?

Reply:
It wasn't. We were only using the Foundation model, which needs a different DTB.

Reply:
OK, I thought we were using this file with our `fvp.xml` manifest.

Reply:
We are, if compiling with `FVP_USE_BASE_PLAT=y`. It was introduced with commit 1bf41f5d9b0d ("fvp: add Trusted Services support") in https://github.com/OP-TEE/build